CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing to Newsboat
+
+Thank you for your interest in contributing to Newsboat! Here are a few ways
+you can help:
+
+1. [Feature Requests](#feature-requests)
+2. [Bug Reports](#bug-reports)
+3. [Patches / Pull Requests](#patches--pull-requests)
+    1. [Testing](#testing)
+    2. [Documentation](#documentation)
+    3. [Translations](#translations)
+
+## Feature Requests
+
+Feature requests should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Ensure
+the feature request was not already reported by searching the issue tracker
+and/or mailing list. Feature requests are filed under the `enhancement` label
+on the [issue tracker](https://github.com/newsboat/newsboat/labels/enhancement).
+
+
+## Reporting Bugs
+
+**Do not open a GitHub issue if the bug is a security vulnerability**. Send an
+email to security@newsboat.org. Please encrypt your emails to 
+[PGP key 4ED6CD61932B9EBE](https://newsboat.org/newsboat.pgp)
+
+Bug reports should be reported in the
+[issue tracker](https://github.com/newsboat/newsboat/issues) or on
+the [mailing list](https://groups.google.com/group/newsboat). Again,
+please ensure the bug was not already reported.
+
+
+## Patches / Pull Requests
+
+Patches can be submitted by sending a 
+[GitHub Pull Request](https://github.com/newsboat/newsboat/pull/new/master) or 
+emailing a patch to the Newsboat 
+[mailing list](https://groups.google.com/group/newsboat). 
+To learn more about patches, see 
+[here](https://www.kernel.org/doc/html/latest/process/submitting-patches.html) and 
+[here](https://github.com/git/git/blob/master/Documentation/SubmittingPatches). To
+learn more about pull requests, 
+see the [GitHub docs](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+
+If you are looking for a place to start contributing, take a look at the issues labeled 
+[good first issue](https://github.com/newsboat/newsboat/labels/good%20first%20issue)
+and [docs](https://github.com/newsboat/newsboat/labels/docs) on the issue tracker.
+[The Newsboat Hacker's Guide](https://github.com/newsboat/newsboat/blob/master/doc/hackers-guide.asciidoc)
```

Comment:
We don't talk about that document :D

More seriously: that document haven't been updated in *ages*. It still has "Newsbeuter" in its title. I wouldn't point people at it.

Reply:
When I was first looking to contribute, I stumbled upon that document and thought it was interesting. If it's out of date I can remove it from here, but it might be a good idea to slowly update it. Having an "architecture and rationale" type page would be useful for contributors imo.

Reply:
I filed https://github.com/newsboat/newsboat/issues/1425 about it. No promises on when it will be fixed, but it got a fighting chance now :)